tests/integration/bond_test.py

```diff
@@ -241,3 +245,20 @@ def test_remove_one_of_the_bond_slaves(eth1_up, eth2_up):
         bond99_cur_state = current_state[INTERFACES][0]
 
     assert bond99_cur_state['link-aggregation']['slaves'] == ['eth2']
+
+
+def test_set_bond_mac_address(eth1_up):
```

Comment:
IMHO it would be nicer to see the MAC tests in their own file to make it easier to see if they cover all device types 
but it is not a blocker for me.

Reply:
I think that we need to choose the main subject of the tests. Having each property as a subject does not fit well IMO.
But perhaps we can centralize all interface type agnostic tests into one test module.